### PR TITLE
Devtooling

### PR DESCRIPTION
**Pull request by stijnvanhoey**
* add setup configuration file
* provide support for [bumpversion](https://github.com/peritus/bumpversion)
* add automatic Travis deployment to Pypi when **push a new tag to master branch**
* add issue template with guidelines for tech and non tech questions
* update development requirements
* add tox for local testing with multiple python versions
* add authors file and add to docs
* add contributing guidelines and add to docs

### CODE REVIEW

**Comment by stijnvanhoey** [comment]
@johanvdw the release to pypi still need to be tested (active on tag push to master)

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+Before you submit a pull request, check that it meets these guidelines:
+
+1. The pull request should include tests.
+2. If the pull request adds functionality, the docs should be updated. Put
+   your new functionality into a function with a docstring, and add the
+   feature to the list in README.rst.
+3. The pull request should work for Python 2.7, 3.5 and 3.6. Check
+   https://travis-ci.org/inbo/niche_vlaanderen/pull_requests
+   and make sure that the tests pass for all supported Python versions.
+
```
Comment (johanvdw) [inline CONTRIBUTING.rst]: I would add:

Note that this are guidelines. If you are stuck while adding functionality - consider doing a pull request anyway, others may help.

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+[bumpversion:file:niche_vlaanderen/__init__.py]
+search = __version__ = '{current_version}'
+replace = __version__ = '{new_version}'
+
+[bdist_wheel]
+universal = 1
+
+[flake8]
+exclude = docs
+
```
Comment (johanvdw) [inline setup.cfg]: we should also exclude the original niche version (orig)

File: CONTRIBUTING.rst (hunk 1 above)
Comment (stijnvanhoey) [reply CONTRIBUTING.rst]: Great suggestion, we should indeed also invite people who are less familiar with the whole github-drill...

File: setup.cfg (hunk 2 above)
Comment (stijnvanhoey) [reply setup.cfg]: indeed ;-)

**Pull request merged**